### PR TITLE
aws-nuke 2.19.0 (new formula)

### PR DESCRIPTION
**Pull request by bodnarbm**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

aws-nuke is a commonly-used cli tool for cleaning up aws accounts (for example, removing resources periodically from developer sandbox accounts). I searched brew for this yesterday when I needed it for a project and could not find it. 

Merging this will also close out https://github.com/rebuy-de/aws-nuke/issues/125.

### CODE REVIEW

File: Formula/aws-nuke.rb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  depends_on "go" => :build
+
+  resource("example_config") do
+    url "https://raw.githubusercontent.com/rebuy-de/aws-nuke/5da46161d4d222216c6928bc4c47406cd9bb4691/config/example.yaml"
+    sha256 "00b476307cf0b9f6ae6bde14af9f237afa9c2225cbb0125171863f43b74fd184"
+  end
+
+  def install
+    major_version = version.major
+    build_xdst="github.com/rebuy-de/aws-nuke/v#{major_version}/cmd"
```
Comment (SMillerDev) [inline Formula/aws-nuke.rb:19]:
```suggestion
    build_xdst="github.com/rebuy-de/aws-nuke/v#{version.major}/cmd"
```

File: Formula/aws-nuke.rb [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    ldflags = %W[
+      -s -w
+      -X #{build_xdst}.BuildVersion=#{version}
+      -X #{build_xdst}.BuildDate=#{time.strftime("%F")}
+      -X #{build_xdst}.BuildHash=#{Utils.git_head}
+      -X #{build_xdst}.BuildEnvironment=homebrew
```
Comment (SMillerDev) [inline Formula/aws-nuke.rb:25]:
```suggestion
      -X #{build_xdst}.BuildEnvironment=#{tap.user}
```

File: Formula/aws-nuke.rb [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    ]
+    with_env(
+      "CGO_ENABLED" => "0",
+    ) do
+      system "go", "build", *std_go_args(ldflags: ldflags), "-o", "#{bin}/aws-nuke", "."
```
Comment (SMillerDev) [inline Formula/aws-nuke.rb:30]: Those are the default 
```suggestion
      system "go", "build", *std_go_args(ldflags: ldflags)
```

File: Formula/aws-nuke.rb [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+    end
+
+    (bash_completion/"aws-nuke").write `#{bin}/aws-nuke completion bash`
+    (fish_completion/"aws-nuke.fish").write `#{bin}/aws-nuke completion fish`
+    (zsh_completion/"_aws-nuke").write `#{bin}/aws-nuke completion zsh`
```
Comment (SMillerDev) [inline Formula/aws-nuke.rb:35]: You should use the safe_read functions, not backticks.

**Comment by bodnarbm** [comment]
Thank you @SMillerDev for the excellent suggestions.

File: Formula/aws-nuke.rb [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+      revision: "357aa44f5a04df9d1faa8c57bb5ce924871a910a"
+  license "MIT"
+  head "https://github.com/rebuy-de/aws-nuke.git", branch: "main"
+
+  depends_on "go" => :build
+
+  resource("example_config") do
+    url "https://raw.githubusercontent.com/rebuy-de/aws-nuke/5da46161d4d222216c6928bc4c47406cd9bb4691/config/example.yaml"
+    sha256 "00b476307cf0b9f6ae6bde14af9f237afa9c2225cbb0125171863f43b74fd184"
+  end
```
Comment (SMillerDev) [inline Formula/aws-nuke.rb:15]: Will this need to be updated for new releases?

**Review by SMillerDev — APPROVED** [review]

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
@bodnarbm, thanks for your contribution to Homebrew! :tada: 🥇

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Pull request closed without merging**